docs/pages/blog/2023-toolpad-beta-announcement.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Toolpad is now in Beta!
+description: After working on it for more than a year, Toolpad is now in the next stage. Read more about it in this announcement.
+date: 2023-07-03T00:00:00.000Z
+authors: ['prakhargupta']
+card: true
+tags: ['Product', 'News']
+---
+
+It's been over a year since we released the first version of Toolpad. Today, we're excited to take the next step on that journey with the release of Toolpad Beta. If you aren't familiar with Toolpad yet, it's an admin panel builder catering to the internal tooling needs of an organization. It's for developers who want to build a functional application quickly. It harnesses the speed of a UI builder for the front-end and closely integrates with your normal development workflow. If this excites you, then read on!
+
+## Why did we decide to build Toolpad?
+
+Toolpad was created to meet the needs of the MUI community. We conducted surveys, did our market research, and concluded to make an internal tool builder because the idea resonates with MUI's mission of empowering developers to build app faster and more efficiently.
+
+Internal tools are software applications that are developed and used within an organization. They can automate tasks, manage data, and help foster collaboration. Internal tools can be a valuable asset for businesses of all sizes, as they can help to improve efficiency, productivity, and communication.
+
+A sub-optimal tool means a waste of time and effort for everyone—maintaining them distracts developers from solving actual business problems. We've all had our fair share of wrestling with lousy internal tools that we hoped we'd never have to use more than once. In recent years, customer-side software has moved quickly and is now flashy and sleek. It's time for internal tools to get this same upgrade.
+
+## What is Toolpad exactly?
+
+Toolpad is an open-source, low-code, drag-and-drop admin builder framework. The primary purpose of Toolpad is to create data-intensive React apps faster. Toolpad simplifies writing back-end logic, connecting to a data source, querying and mutating data, and deploying an app. It can quickly convert an API, script, or SQL query into a web UI.
```

Comment:
Would we really call this a framework?

```suggestion
Toolpad is an open-source, low-code, drag-and-drop admin builder. The primary purpose of Toolpad is to create data-intensive React apps faster. Toolpad simplifies writing back-end logic, connecting to a data source, querying and mutating data, and deploying an app. It can quickly convert an API, script, or SQL query into a web UI.
```

Reply:
Removing framework, maybe it is not the right term to define Toolpad.